CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ---
 
 ## master
+* HMPP support
```

Comment:
Should the `iosSimulatorArm64` slice be mentioned explicitly too?

Reply:
Done